integration_test/validate_metadata_test.go

```diff
@@ -28,6 +31,27 @@ func TestValidateMetadataOfThirdPartyApps(t *testing.T) {
 	}
 }
 
+func TestMissingMetadataForThirdPartyApps(t *testing.T) {
+	parentDirectory := "third_party_apps_data/applications"
+	dirs, err := thirdPartyDataDir.ReadDir(parentDirectory)
+
+	if err != nil {
+		t.Error(err)
+	}
+
+	for _, dir := range dirs {
+		dirName := dir.Name()
```

Comment:
This variable is only used once, and doesn't buy you anything. I'd inline it.

Reply:
Changed in: [8e4de5c](https://github.com/GoogleCloudPlatform/ops-agent/pull/793/commits/8e4de5c7b435508e229c3e0ab8dbda8c5a3b67b7)